### ISSUE TITLE
used only once warnings [rt.cpan.org #65193]

### ISSUE THREAD

**Issue by ingydotnet** [ingydotnet/yaml-pm#109]
https://rt.cpan.org/Ticket/Display.html?id=65193

```
----- BEGIN CODE -----
use strict;
use warnings;

use YAML qw( );

local $YAML::UseCode = 1;
my $s = YAML::Load(<<'__EOI__');

---
- !!perl/code |
  {
      print "Hello World\n";
  }
__EOI__

$s->[0]->();
----- END CODE -----

----- BEGIN OUTPUT -----
Name "YAML::UseCode" used only once: possible typo at x.pl line 6.
Hello World
----- END OUTPUT -----

That warning is spurious. Please add the following to YAML to silence it:

----- BEGIN CODE -----
use vars qw( $UseCode );
----- END CODE -----

The same applies to the other global config variables.
```

**Comment by perlpunk** [comment]
Created PR #171

**Comment by perlpunk** [comment]
Fixed by #171

**Comment by perlpunk** [comment]
released 1.20_002
